src/pages/ThorChainLP/components/LpType.tsx

```diff
@@ -120,21 +127,30 @@ export const LpType = ({ assetId, opportunityId, onAsymSideChange }: DepositType
       const radio = getRadioProps({ value: option.value })
       const optionAssetIds = makeAssetIdsOption(option.value as AsymSide | 'sym')
 
+      const isDisabled = !!side && opportunityType !== 'sym' && option.value === 'sym'
```

Comment:
rationale: if we make `side` (`opportunityType`) required then it will always be defined, and `opportunityType` checks should be enough regardless

```suggestion
      const isDisabled = opportunityType !== 'sym' && option.value === 'sym'
```

Reply:
q: as a user, can I e.g deposit RUNE asym, then withdraw asset asym, or does this warrant another check?

Reply:
https://dev.thorchain.org/concepts/memos.html#withdraw-liquidity

```
A withdrawal can be either dual-sided (withdrawn based on pool's price) or entirely single-sided (converted to one side and sent out).

Ex: WITHDRAW:POOL:10000 — dual-sided 100% withdraw liquidity. If a single-address position, this withdraws single-sidedly instead
```

This check is to disable the sym withdraw if you deposited as an asym side, I changed it to a `isWithdraw` prop and removed the `side` prop

![image](https://github.com/user-attachments/assets/3d3161d3-8597-46d2-853a-10fb8d6e7e9d)

In [3f46c39](https://github.com/shapeshift/web/pull/7351/commits/3f46c39c62ae8b335139b436a73c7c22405becd5)

Reply:
tyvm for the clarification!